public/app/plugins/datasource/prometheus/querybuilder/components/MetricSelect.tsx

```diff
@@ -22,7 +22,7 @@ export interface Props {
   labelsFilters: QueryBuilderLabelFilter[];
 }
 
-const MAX_NUMBER_OF_RESULTS = 1000;
```

Comment:
why do you have to refactor stuff that's outside the scope of this PR? smh, but it is a nicer name

Reply:
Oh yeah, we're exporting it now, so this is in scope, ignore me!